consequence_prediction/vep_mapping_pipeline/consequence_mapping.py

```diff
@@ -15,22 +15,13 @@
 from retry import retry
 
 parser = argparse.ArgumentParser(description=__doc__)
-parser.add_argument(
-    '--enable-distant-querying', action='store_true',
-    help='Enables a second iteration of querying VEP for distant gene variants, which is disabled by default'
-)
-parser.add_argument(
-    '--report-distance', action='store_true',
-    help='Report a distance to the gene for upstream/downstream gene variants. Disabled by default'
-)
 
 logging.basicConfig()
 logger = logging.getLogger('consequence_mapping')
 logger.setLevel(logging.INFO)
 
-# The "distance to the nearest gene" parameters, used to query VEP in first and second iterations, respectively.
+# The "distance to the nearest gene" parameters, used to query VEP.
 VEP_SHORT_QUERY_DISTANCE = 5000
-VEP_LONG_QUERY_DISTANCE = 500000
```

Comment:
What was the reason behind no longer querying VEP twice if the short distance doesn't find any gene? Just curious